docs/source/_static/css/custom.js

```diff
@@ -0,0 +1,20 @@
+// _static/js/custom.js
+document.addEventListener("DOMContentLoaded", function() {
+    var closeButton = document.getElementById("close-banner");
+    var banner = document.getElementById("announcement-banner");
+
+    var bannerClosed = localStorage.getItem('bannerClosed');
```

Comment:
how come even if using localstroage the value is still not persisted across pages?